react/src/components/Widget/Widget.tsx

```diff
@@ -289,7 +289,9 @@ export const Widget: React.FC<WidgetProps> = props => {
       url = prefixedAddress + (query.length ? `?${query.join('&')}` : '');
       setUrl(url);
     } else {
-      if (!isFiat && currencyObj) {
+      const notZeroValue: boolean =
+        currencyObj?.satoshis !== undefined && currencyObj.satoshis > 0;
```

Comment:
`currencyObj?.satoshis` I'm not familiar with what the '?' does here. Able to explain?